tests/MongoDB.Driver.Core.Tests/Core/Clusters/ClusterTests.cs

```diff
@@ -32,6 +32,7 @@
 using MongoDB.Bson.TestHelpers.XunitExtensions;
 using System.Reflection;
 using MongoDB.Driver.Core.Bindings;
+using MongoDB.Bson.TestHelpers;
```

Comment:
There are a number of namespaces(added in previous tickets) which are not used in this class. They can be removed if you don't mind

Reply:
Done.